<!--REQUEST_ID=6d897623-5518-4508-a653-3bd6dec52984-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47933&attempt_id=6b43d762-2ce2-4358-abd4-144404275db3)

Attempted: vault

<details><summary>Partial log (click to expand)</summary><p>

```
==> Checking that build is using go version >= 1.10...
==> Using go version 1.11...
==> Removing old directory...
==> Building...
No valid platforms to build for. If you specified a value
for the 'os', 'arch', or 'osarch' flags, make sure you're
using a valid value.
make: *** [Makefile:34: dev] Error 1
builder for '/nix/store/g0xkf1a2c3iq1f7rrhfy6s44njcfmqbi-vault-0.11.2.drv' failed with exit code 2
error: build of '/nix/store/g0xkf1a2c3iq1f7rrhfy6s44njcfmqbi-vault-0.11.2.drv' failed
```
</p></details>

--- a/pkgs/tools/security/vault/default.nix
+++ b/pkgs/tools/security/vault/default.nix
@@ -11,13 +11,13 @@ let
   };
 in stdenv.mkDerivation rec {
   name = "vault-${version}";
-  version = "0.11.1";
+  version = "0.11.2";
 
   src = fetchFromGitHub {
     owner = "hashicorp";
     repo = "vault";
     rev = "v${version}";
-    sha256 = "1ydnb9z6rd5ck6wza5ir6927xq375i1a9zh5p2xanp29ly6ijiiz";
+    sha256 = "0lckpfp1yw6rfq2cardsp2qjiajg706qjk98cycrlsa5nr2csafa";
   };
 
   nativeBuildInputs = [ go gox removeReferencesTo ];

